README.md

```diff
@@ -108,6 +108,16 @@ If you have verified this command on Windows, we invite you to submit a PR to in
 
 In addition to local development with Python `venv`, it is also possible to use the devcontainer found in the root of the project.
 
+### Using Docker
+
+A barebones [dockerfile](./dockerfile) is supplied to run the site within a local Docker Container through [Docker Desktop](https://docs.docker.com/desktop/) - a simple, free, way to easily setup Docker and Python without leaving beyond installations, modifying your underlying Operating System, and changing Environment Variables.
+
+1. Make sure Docker Desktop is running and started
+1. Build the Docker Image from the root directory with the command: `docker build -t wtd .`
```

Comment:
@plaindocs - I tweaked the recommended build per your prior comment here: https://github.com/writethedocs/www/pull/2379#discussion_r2129010522 - loved that suggestion - thanks!